anthos/deploy_admin_ws.sh

```diff
@@ -1,5 +1,63 @@
 #!/bin/bash
 
+# print a message to stderr, prefixed by HOSTNAME
+function note() {
+  echo 1>&2 "$HOSTNAME: $*"
+}
+
+# print the given command to stderr, run it, and exit verbosely if it fails.
+function xrun() {
+  vrun "$@" && return 0
+
+  local xstat=$?
+  note "Cmd $1 failed, exit $xstat"
+  exit "$xstat"
+}
+
+# print the given command to stderr, run it.  return the status
+function vrun() {
+  note "+ $@"
+  "$@"
+}
+
+# fetch the vcenter pem from the givem host:port, and print it to stdout.
+function fetch_pem() {
+djfong@djfong-macbookair2$ cat !$
```

Comment:
It looks like some copy/paste screen capture made it into the script. (`djfong@djfong-macbookair2$ `)

I received the following when testing this:
```
null_resource.anthos_deploy_workstation[0]: Provisioning with 'remote-exec'...
...
null_resource.anthos_deploy_workstation[0] (remote-exec): edge-gateway01: + govc pool.create */Anthos
null_resource.anthos_deploy_workstation[0] (remote-exec): edge-gateway01: # succeeded
null_resource.anthos_deploy_workstation[0] (remote-exec): /root/anthos/deploy_admin_ws.sh: line 82: syntax error: unexpected end of file
```

Reply:
thanks, i will fix it.